hadoop-ozone/dist/src/main/compose/ozone/docker-compose.yaml

```diff
@@ -26,15 +26,15 @@ services:
       command: ["/opt/hadoop/bin/ozone","datanode"]
       env_file:
         - ./docker-config
-   ozoneManager:
+   om:
       image: apache/hadoop-runner
       privileged: true #required by the profiler
       volumes:
          - ../..:/opt/hadoop
       ports:
          - 9874:9874
       environment:
-         ENSURE_OM_INITIALIZED: /data/metadata/ozoneManager/current/VERSION
+         ENSURE_OM_INITIALIZED: /data/metadata/om/current/VERSION
```

Comment:
Here we have changed to om, will be this be same as service name when path is created?

Reply:
changed this for consistency purpose. IMO path can be independent of service name.